test/broadcasting.jl

```diff
@@ -148,7 +148,11 @@ end
     @test df[:, 2:end] == refdf[:, 2:end]
 
     dfv = @view df[1:2, 2:end]
-    @test_throws ArgumentError dfv[!, 1] .+= 1
+    dfv[!, 1] .+= 100
+    @test df.x2 == [104.5, 105.5, 6.5]
+    df.x1 -= [1, 1, 1]
+    df.x2 -= [100, 100, 0]
+    @test df == refdf
```

Comment:
Why not simply compare `df` with a literal `DataFrame`?

Reply:
ok - changed